ivy/functional/frontends/torch/tensor.py

```diff
@@ -419,6 +419,10 @@ def tril(self, diagonal=0):
     def index_select(self, dim, index):
         return torch_frontend.index_select(self._ivy_array, dim, index)
 
+    @with_unsupported_dtypes({"1.11.0 and below": ("bfloat16", "float16")}, "torch")
+    def clamp(self, min=None, max=None, *, out=None):
+        return torch_frontend.clamp(self._ivy_array, min=min, max=max, out=out)
```

Comment:
I suggest handling the case when `max < min` here, otherwise return the `torch_frontend.clamp(...)` and check whether it has worked or not.